gst/nnstreamer/registerer/nnstreamer.c

```diff
@@ -67,9 +67,11 @@
 #include <tensor_transform/tensor_transform.h>
 #include <tensor_if/gsttensorif.h>
 #include <tensor_rate/gsttensorrate.h>
+#if (ENABLE_NNS_QUERY == 1)
```

Comment:
IMO, ```tensor_query/tensor_query_*.h``` can hide this line and the following ```#endif```.

Reply:
Oh.. Interesting.
But the mqtt-hybrid (using `nnsquery) code will be separated from tensor_query soon, so it will be reverted.